### PR TITLE
The Az'Marian comic

### PR DESCRIPTION
**Pull request by Spookat**
Added a new Tajara comic to the general loadout - The Az'Marian.

Features 8 different issues visible on extended description.

### CODE REVIEW

**Comment by Spookat** [comment]
!review

**Review by Alberyk — APPROVED** [review]

File: code/game/objects/items/toys.dm [hunk 1]
```diff
@@ -333,6 +333,65 @@
 	icon_state = "comicoutlandish"
 	item_state = "comicoutlandish"
 
+/obj/item/toy/comic/azmarian
+	name = "az'marian comic"
+	desc = "A comic book series revolving around a superhero from Az'Mar, popular in the New Kingdom. Sensing great sorrow that disturbed \
+	his slumber from the world of Adhomai, the Az'Marian flew to the planet, finding that the peaceful ways of the Ancient Tajara were \
+	completely disrupted by war and rebellion. The superhero joins forces with the descendant kingdom of the Ancient Tajara, helping \
+	fend off its enemies that threaten to push the Tajara towards doom. Critics often note that most of the villains are Zhan'Khazan, \
+	while most supporting characters are Njarir or Hharar."
```
Comment (SleepyGemmy) [inline code/game/objects/items/toys.dm:342]:
```suggestion
	desc = "A comic book series revolving around a superhero from Az'Mar, popular in the New Kingdom.
```
use `desc_extended` for this.

File: code/game/objects/items/toys.dm [hunk 2]
```diff
@@ -333,6 +333,65 @@
...
+			by Sham'tyrs, facing freezing winds and snow, the two will trek to reach the shrine in hope of recovery. Bandits encampments, a village in need of help, \
+			a chance encounter with the S'Mara, and a shape-shifting spy amongst the Sham'tyrs, will the heroes make their way in one piece?"
+		if(8)
+			desc_extended = "Issue 8 - The Shrine - In this issue, the hero and his loyal but dim-witted sidekick Murr'zhul meet the Parivara at the ancient shrine. \
+			Tended to by the Sun Sister with the blessing of Messa, the Az'Marian manages to recover from his affliction over the course of a week. He learns of an \
+			evil plan staged by a group of Cae'rrin living in caves nearby to desecrate the shrine and eat its attendants, and the two embark on a journey to put a \
+			stop to this plan. Already aware of the Az'Marian's presence from their spy, they begin to set up traps and occult rituals to prevent him from even reaching \
+			the caves. Forces of Raskara, shape-shifting creatures, dastardly traps and true evil lurking within, the Az'Marian and his sidekick will have to use their \
+			newly regained powers to their fullest."
```
Comment (SleepyGemmy) [inline code/game/objects/items/toys.dm:393]: would be preferable if this was something the player could choose in the loadout, or when they spawn.

Comment (alsoandanswer) [reply code/game/objects/items/toys.dm:389]: would bloat the loadout section too much

**Review by alsoandanswer — APPROVED** [review]

**Comment by SleepyGemmy** [comment]
do not merge as the PR awaits some changes.

**Review by SleepyGemmy — APPROVED** [review]

**Pull request merged**